src/components/button-pill/ButtonPill.jsx

```diff
@@ -1,8 +1,12 @@
 import React from "react";
 
 const ButtonPill = ({text}) => {
+    const handleClick = () => {
+        console.log('CityBoy Villains');
+    }
+
```

Comment:
ah so you're super close! it's great that you got `onClick` working but it needs to be a prop!

just like how we have `text as a prop, we need to add `onClick` as a prop too, and that's all we'd have to change! everything in the return function looks good to me